routes/items.js

```diff
@@ -24,15 +24,16 @@ router.post( '/', ( request, response ) => {
     .then( result => response.redirect( '/items' ))
 })
 
-router.post( '/:id/completed', ( request, response ) => {
+router.post( '/:id', ( request, response ) => {
   const Item = request.app.get( 'models' ).Item
 
   const { id } = request.params
-  const { completed } = request.body
+  const { title, description, completed } = request.body
 
-  Item.update({ completed }, { where: { id, user_id: request.user.id }})
+  Item.update( { title, description, completed }, { where: { id, user_id: request.user.id }})
```

Comment:
I don't think we can do this, because when a field is not provided, it would be undefined, which might overwrite a value in the DB?  See my comment about validation.